modules/react/form-field/stories/visual-testing/stories_Radio.tsx

```diff
@@ -97,13 +97,23 @@ export const RadioStates = () => (
         ]}
       >
         {props => (
-          <FormField
-            useFieldset={true}
-            hintText={typeof props.error !== 'undefined' ? hintText : undefined}
-            hintId={hintId}
-            {...props}
-          >
-            {testGroup}
+          <FormField as="fieldset" {...props}>
+            <FormField.Label>{props.label}</FormField.Label>
+            {props.orientation === 'horizontal' ? (
+              <FormField.Container>
+                {testGroup}
+                {typeof props.error !== 'undefined' ? (
+                  <FormField.Hint>hintText</FormField.Hint>
+                ) : undefined}
```

Comment:
what other possible types for `error` prop? In old version it was reasonable to have this validation to avoid type inconsistency in prop value. But in your case you can keep it as:

```
props.error && (
  <FormField.Hint>hintText</FormField.Hint>
)
```

except you want to show hint on `null` or `false` value of error

Reply:
Suggestion: Don't you want to move validation outside this block and keep it only for wrapper? 

like:
`const Wrapper = props.error ? FormField.Hint : React.Component`
and then

```
             <Wrapper>
                {testGroup}
                {props.error && (
                  <FormField.Hint>hintText</FormField.Hint>
                )}
              </Wrapper>
```